### DISCUSSION TITLE
How to update the edge?

### DISCUSSION THREAD

**Discussion by m-tatsuya** [bcakmakoglu/vue-flow#180] [Q&A] [answered]
I referred to "Updatable Edge" in Examples, but couldn't update Edge.
I placed the Source of Examples in my project and checked it, but I couldn't update Edge as well.

Do you need more action than in the Examples source?
(Reference: https://vueflow.dev/examples/updatable-edge.html)

Thanks.

**Comment by bcakmakoglu** [comment]
Hey! 
Thanks for using Vue Flow.
That seems to be a bug. I will deploy a fix during the day in a new release!

**Reply by bcakmakoglu** [reply]
Check #181 for the progress.

**Comment by bcakmakoglu** [comment, marked as answer]
Fixed with release 0.4.19

**Reply by m-tatsuya** [reply]
Thank you so much!
